project/Build.scala

```diff
@@ -1092,6 +1093,40 @@ object Build {
       fatalWarningsSettings ++ Seq(name := "Scala.js JUnit test runtime")
   ).withScalaJSCompiler.dependsOn(testInterface)
 
+  val commonJUnitTestOutputsSettings = commonSettings ++ fatalWarningsSettings ++ Seq(
+      publishArtifact in Compile := false,
+      parallelExecution in Test := false,
+      unmanagedSourceDirectories in Test +=
+          baseDirectory.value.getParentFile / "shared/src/test/scala",
+      testOptions += Tests.Argument(TestFrameworks.JUnit, "-v", "-a", "-s"),
+      test in Test := {
+        val _ = {
+          (testOnly in Test).toTask(" *Assertions").result.value
+        }
```

Comment:
And, actually, why not manipulating `testFilter` instead, as suggested by @gzm0?